android-sdk/src/main/java/org/sagebionetworks/bridge/android/data/ArchiveInfo.java

```diff
@@ -0,0 +1,93 @@
+package org.sagebionetworks.bridge.android.data;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+import com.google.gson.annotations.SerializedName;
+
+import org.joda.time.DateTime;
+
+import java.util.List;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+/**
+ * This is the model for info.json
+ */
+@SuppressWarnings("unused")
```

Comment:
This class is used for GSON which uses reflection to access the fields, so this suppresses IntelliJ's warnings about fields that are never set or retrieved.

Another option is to tell IntelliJ to ignore for fields that are annotated with `@SerializedName`. I think this setting can be done project level and shared.